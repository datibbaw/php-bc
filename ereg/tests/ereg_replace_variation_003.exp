*** Testing ereg_replace() : usage variations ***
Error: 8 - Undefined variable: undefined_var, %s(64)
Error: 8 - Undefined variable: unset_var, %s(67)

Arg value 0 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(1) "0"

Arg value 1 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(9) "new value"

Arg value 12345 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(13) "new value2345"

Arg value -2345 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(5) "-2345"

Arg value 10.5 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(12) "new value0.5"

Arg value -10.5 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(13) "-new value0.5"

Arg value 101234567000 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(28) "new value0new value234567000"

Arg value 1.07654321E-9 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(29) "new value.0765432new valueE-9"

Arg value 0.5 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(3) "0.5"
Error: 8 - Array to string conversion, %sereg_replace_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace() expects parameter 3 to be string, array given, %s(74)
NULL
Error: 8 - Array to string conversion, %sereg_replace_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace() expects parameter 3 to be string, array given, %s(74)
NULL
Error: 8 - Array to string conversion, %sereg_replace_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace() expects parameter 3 to be string, array given, %s(74)
NULL
Error: 8 - Array to string conversion, %sereg_replace_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace() expects parameter 3 to be string, array given, %s(74)
NULL
Error: 8 - Array to string conversion, %sereg_replace_variation_003.php(%d)

Arg value Array 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace() expects parameter 3 to be string, array given, %s(74)
NULL

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(0) ""

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(0) ""

Arg value 1 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(9) "new value"

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(0) ""

Arg value 1 
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(9) "new value"

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(0) ""

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(0) ""

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(0) ""
Error: 4096 - Object of class stdClass could not be converted to string, %s(73)

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
Error: 2 - ereg_replace() expects parameter 3 to be string, object given, %s(74)
NULL

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(0) ""

Arg value  
Error: 8192 - Function ereg_replace() is deprecated, %s(74)
string(0) ""
Done