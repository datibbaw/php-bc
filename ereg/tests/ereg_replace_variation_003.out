*** Testing ereg_replace() : usage variations ***
Error: 8 - Undefined variable: undefined_var, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_003.php(64)
Error: 8 - Undefined variable: unset_var, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_003.php(67)

Arg value 0 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(1) "0"

Arg value 1 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(9) "new value"

Arg value 12345 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(13) "new value2345"

Arg value -2345 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(5) "-2345"

Arg value 10.5 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(12) "new value0.5"

Arg value -10.5 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(13) "-new value0.5"

Arg value 101234567000 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(28) "new value0new value234567000"

Arg value 1.07654321E-9 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(29) "new value.0765432new valueE-9"

Arg value 0.5 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(3) "0.5"
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_003.php(73)

Arg value Array 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
array(0) {
}
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_003.php(73)

Arg value Array 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
array(1) {
  [0]=>
  string(1) "0"
}
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_003.php(73)

Arg value Array 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
array(1) {
  [0]=>
  string(9) "new value"
}
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_003.php(73)

Arg value Array 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
array(2) {
  [0]=>
  string(9) "new value"
  [1]=>
  string(1) "2"
}
Error: 8 - Array to string conversion, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_003.php(73)

Arg value Array 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
array(2) {
  ["color"]=>
  string(3) "red"
  ["item"]=>
  string(3) "pen"
}

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(0) ""

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(0) ""

Arg value 1 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(9) "new value"

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(0) ""

Arg value 1 
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(9) "new value"

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(0) ""

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(0) ""

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(0) ""
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/tests/ereg_replace_variation_003.php(73)

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
Error: 4096 - Object of class stdClass could not be converted to string, /Users/tjerk/Documents/php-bc/ereg/ereg.php(137)
string(0) ""

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(0) ""

Arg value  
Error: 16384 - Function ereg_replace() is deprecated, /Users/tjerk/Documents/php-bc/ereg/ereg.php(135)
string(0) ""
Done